app/assets/javascripts/components/overview/my_articles/components/Categories/List/Assignment/Header/Actions/MarkAsIncompleteButton.jsx

```diff
@@ -1,36 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { updateAssignmentStatus, fetchAssignments } from '../../../../../../../../../actions/assignment_actions';
```

Comment:
This much relative directory navigation is very hard to understand visually. Can you use an `@` path shortcut instead (as defined in jsconfig.json and used for example in `article.jsx`)?

Reply:
Okay. I added an actions path and replaced my imports with it. You're right, the imports look much nicer now.